### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,21 +1,70 @@
...
-<!-- default file list -->
-*Files to look at*:
 
-* [HomeController.cs](./CS/Q515371/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/Q515371/Controllers/HomeController.vb))
-* [TreeListVirtualModeHelper.cs](./CS/Q515371/Models/TreeListVirtualModeHelper.cs) (VB: [TreeListVirtualModeHelper.vb](./VB/Q515371/Models/TreeListVirtualModeHelper.vb))
-* [_TreeListPartial.cshtml](./CS/Q515371/Views/Home/_TreeListPartial.cshtml)
-* [Index.cshtml](./CS/Q515371/Views/Home/Index.cshtml)
-<!-- default file list end -->
-# TreeList - How to load data from a database in virtual mode and implement drag-and-drop
+# Tree List for ASP.NET MVC - How to load data from a database in virtual mode and implement drag-and-drop functionality
```
Comment (albertov05) [inline Readme.md:6]: If "Tree List" refers to the extension name, then shouldn't it be "TreeList"?

File: Readme.md [hunk 2]
```diff
@@ -1,21 +1,70 @@
...
+
+Use the [BindToVirtualData](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.TreeListExtension.BindToVirtualData(DevExpress.Web.Mvc.TreeListVirtualModeCreateChildrenMethod-DevExpress.Web.Mvc.TreeListVirtualModeNodeCreatingMethod)) method to implement virtual mode for the [TreeList](https://docs.devexpress.com/AspNetMvc/13765/components/tree-list) extension. The method requires two following delegate methods as parameters:
```
Comment (albertov05) [inline Readme.md]:
```suggestion
Use the [BindToVirtualData](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.TreeListExtension.BindToVirtualData(DevExpress.Web.Mvc.TreeListVirtualModeCreateChildrenMethod-DevExpress.Web.Mvc.TreeListVirtualModeNodeCreatingMethod)) method to implement virtual mode for the [TreeList](https://docs.devexpress.com/AspNetMvc/13765/components/tree-list) extension. The method requires the following two delegate methods as parameters:
```

File: Readme.md [hunk 3]
```diff
@@ -1,21 +1,70 @@
...
+
+* `createChildrenMethod` - handle the method to create a list of child nodes owned by the processed node. Assign the list to the event parameter’s `Children` property. The `NodeObject` property returns the node currently being processed. If the event is raised for the root node, the `NodeObject` property returns `null` (Nothing for VB).
```
Comment (albertov05) [inline Readme.md]:
```suggestion
* `createChildrenMethod` - Handle this method to create a list of child nodes owned by the processed node. Assign the list to the event parameter’s `Children` property. The `NodeObject` property returns the node currently being processed. If the event is raised for the root node, the `NodeObject` property returns `null` (Nothing for VB).
```

File: Readme.md [hunk 4]
```diff
@@ -1,21 +1,70 @@
...
+    Employee parentEmployee = e.NodeObject as Employee;
+    if (parentEmployee == null) {
+        e.Children = nwd.Employees.Where(empl=>empl.ReportsTo==null).ToList();
+    }
+    else {
+        e.Children = (from empl in nwd.Employees where empl.ReportsTo == parentEmployee.EmployeeID select empl).ToList();
+    }          
+  }
+  ```
+* `nodeCreatingMethod` - handle the method to initialize a node in a tree. You should specify a node key value (the `NodeKeyValue` property) and cell values. If processed node has no child nodes, set the `IsLeaf` property to `true`.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
* `nodeCreatingMethod` - Handle this method to initialize a node in a tree. You should specify a node key value (the `NodeKeyValue` property) and cell values. If the processed node has no child nodes, set the `IsLeaf` property to `true`.
```

File: Readme.md [hunk 5]
```diff
@@ -1,21 +1,70 @@
...
+      e.SetNodeValue("Address", empl.Address);
+      e.SetNodeValue("City", empl.City);
+      e.SetNodeValue("HireDate", empl.HireDate);
+      e.SetNodeValue("BirthDate", empl.BirthDate);
+      e.SetNodeValue("ReportsTo", empl.ReportsTo);
+      e.SetNodeValue("Title", empl.Title);
+  }
+  ```
 
+These methods are  implemented in the `TreeListVirtualModeHelper` class.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
These methods are implemented in the `TreeListVirtualModeHelper` class.
```

File: Readme.md (hunk 1 above)
Comment (LanaDX) [reply Readme.md:6]: We use Tree List in docs and site as a marketing name. I think it is ok in an example name

Comment (albertov05) [reply Readme.md:6]: Okay, I understand. Thank you!

**Pull request merged**